src/main/java/frc/robot/RobotContainer.java

```diff
@@ -132,6 +155,14 @@ private void setSmartDashboardSubsystems() {
     SmartDashboard.putData(lowPressureSubsystem);
   }
 
+  private void debugJoystics() {
+    SmartDashboard.putNumber("DriverJoyR", driverController.getY(GenericHID.Hand.kRight));
+    SmartDashboard.putNumber("DriverJoyL", driverController.getY(GenericHID.Hand.kLeft));
+
+    SmartDashboard.putNumber("MechJoyR", mechanismController.getY(GenericHID.Hand.kRight));
+    SmartDashboard.putNumber("MechJoyL", mechanismController.getY(GenericHID.Hand.kLeft));
+  }
+
```

Comment:
```suggestion
```

related to above, but it was just here to debug the drive, so we can probably get rid of this.